Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_fun.yml

```
@@ -40,6 +40,14 @@
       - DrinkSpaceGlue
   - type: TrashOnSolutionEmpty
     solution: drink
+  - type: BoneGel # 🌟Starlight🌟
+  - type: SurgeryTool # 🌟Starlight🌟
+    successRate: 0.4 # really not ideal, but in a pinch...
+    speed: 1.5
+    startSound:
+      path: /Audio/Items/Medical/ointment_begin.ogg
+    endSound:
+      path: /Audio/Items/Medical/ointment_end.ogg
```

```suggestion
  # 🌟Starlight-start🌟
  - type: BoneGel
  - type: SurgeryTool
    successRate: 0.4 # really not ideal, but in a pinch...
    speed: 1.5
    startSound:
      path: /Audio/Items/Medical/ointment_begin.ogg
    endSound:
      path: /Audio/Items/Medical/ointment_end.ogg
  # 🌟Starlight-end🌟
```

Pls use such format to multiline changes, so when we have conflict, we can see start and end of our changes

---

Right... In most places here I'm adding the entire thing, not just messing with the numbers like for the chainsaw. Will adjust.